docs/pages/bare/updating-your-app.mdx

```diff
@@ -105,6 +105,24 @@ In **Expo.plist**, you'll need to add the following, replacing `your-channel-nam
 </dict>
 ```
 
+This configuration is also generated by `expo prebuild` if you set the `requestHeaders` property in your **app.json**: 
+
+```json
+{
+  "expo": {
+  /* @hide ... your existing configuration */ /* @end */
+    "updates": {
+      /* @hide ... your existing configuration */ /* @end */
+      "requestHeaders": {
+        "expo-channel-name": "your-channel-name"
+      }
+      /* @hide ... your existing configuration */ /* @end */
+    }
+  /* @hide ... your existing configuration */ /* @end */
+  }
+}
+```
```

Comment:
```suggestion
```json app.json
{
  "expo": {
  /* @hide ... your existing configuration */ /* @end */
    "updates": {
      /* @hide ... your existing configuration */ /* @end */
      "requestHeaders": {
        "expo-channel-name": "your-channel-name"
      }
      /* @hide ... */ /* @end */
    }
  /* @hide ... */ /* @end */
  }
}
```
```

Let's add the file name to the snippet, so we can display it with "Copy" button.

Reply:
Done